tests/upgrades/test_bookmarks.py

```diff
@@ -0,0 +1,96 @@
+"""Test for bookmark related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+from upgrade_tests import post_upgrade
+from upgrade_tests import pre_upgrade
+
+from robottelo.constants import BOOKMARK_ENTITIES
+from robottelo.test import APITestCase
+
+
+BOOKMARK_ENTITIES = list(BOOKMARK_ENTITIES)
+
+
+class ScenarioPositiveBookmark(APITestCase):
+    """
+    Created Bookmarks in pre upgrade should be unchanged after post upgrade
+    and verify the create and delete operation.
+
+    Test Steps:
+
+        1. Before Satellite upgrade:
+        2. Create set of bookmarks using available bookmark entities
```

Comment:
We are creating bookmark entities from available data and not from available bookmark entities